### ISSUE TITLE
Wrong Beat Saber Version: 1.0.1

### ISSUE THREAD

**Issue by cryptk** [the-expanse/SideQuest#61]
Hello,

When I try and sync Beat Saber onto my quest to add some custom songs, after performing the backup it states that I have the wrong Beat Saber version of 1.0.1... I'm guessing this means that something about how SideQuest patches Beat Saber needs to be updated...

Is there a workaround that I can do until SideQuest is updated?

**Comment by amydionne** [comment]
I have have got the same issue.

**Comment by cryptk** [comment]
Yep, looks like 1.0.1 literally just dropped for the Quest today...

https://www.reddit.com/r/OculusQuest/comments/bwsndn/new_beat_saber_patch/

Considering that trishume (dev of QuestSaberPatch, which I think is what SideQuest uses to patch BeatSaber) posted in that reddit post... it seems he is already aware.

**Comment by amydionne** [comment]
Ugh, I guess that prob what threw off my whole setup so when I deleted the folder it went downhill from there. I have been wrecking myself all day trying to fix it... Ty for the update

**Comment by ChrisDoesStuff2** [comment]
Do we have an ETA for the update? Only asking as for some reason the only backup I have is the latest version of the APK.

**Comment by amydionne** [comment]
> Do we have an ETA for the update? Only asking as for some reason the only backup I have is the latest version of the APK.

Same,I deleted the folder thus deleting the old backup before I found out it was the update that had been the problem.

**Comment by ATechAdventurer** [comment]
A fix is coming momentarily

**Comment by ChrisDoesStuff2** [comment]
Amazing! Is that why the source code keeps updating?

**Comment by ATechAdventurer** [comment]
@ChrisDoesStuff2 @amydionne https://github.com/the-expanse/SideQuest/releases/tag/0.2.1-alpha.1

**Comment by amydionne** [comment]
> @ChrisDoesStuff2 @amydionne https://github.com/the-expanse/SideQuest/releases/tag/0.2.1-alpha.1

wheres the mac link?

**Comment by ATechAdventurer** [comment]
`SideQuest-darwin-x64.zip` is the mac build

**Comment by zaccolley** [comment]
I installed the new version but I'm getting `Install failed` followed by this error message. I've reset my base APK as it suggests and get the same:

```
System.ApplicationException: Trying to patch different version of code at LibSaberPatch.Apk.PatchSignatureCheck() in /home/travis/build/trishume/QuestSaberPatch/LibSaberPatch/Apk.cs:line 91 at jsonApp.Program.RunInvocation(Invocation inv) in /home/travis/build/trishume/QuestSaberPatch/jsonApp/Program.cs:line 63
```

Happy to move this to another issue if no one else is getting this.

Also thank you for the quick response definitely don't worry too much about this, there's no rush from me!

**Comment by amydionne** [comment]
> `SideQuest-darwin-x64.zip` is the mac build

Oh lol pardon my ignorance.

**Comment by ATechAdventurer** [comment]
@zaccolley thank you for reporting that It looks like our dependeny tools don't yet support 1.0.1 like due to changes to the internal structure.

**Comment by zaccolley** [comment]
Cool, let me know if you want me to try anything else out later. :+1:

**Comment by mikeew83** [comment]
Mac App is giving error will not run.

**Comment by amydionne** [comment]
I am getting the error to make sure that this application works with my
version of mac osx, it also updated today.

**Comment by ChrisDoesStuff2** [comment]
On windows here, unless I'm being very stupid dont quite understand the latest update? Obviously the incorrect version has gone but fails to patch at the very start.

**Comment by davestationVR** [comment]
I'm having trouble running the new Windows exe, won't open due to missing ffmpeg.dll? Didn't have this issue with the previous version.

**Comment by lazerwalker** [comment]
Having the same issue others are mentioning on MacOS, looks like it's also missing ffmpeg. Stack trace here: https://gist.github.com/lazerwalker/3725761763af0a127c8f426e34e0e20f

**Comment by XxCreepdawgxX** [comment]
Is Darwin the right asset, or is there no OSX build in this release?

**Comment by amydionne** [comment]
> Is Darwin the right asset, or is there no OSX build in this release?

I asked he said Darwin was OSX...

**Comment by lazerwalker** [comment]
Yes, Darwin is macOS. Darwin is the name of the underlying FreeBSD-based kernel powering macOS.

**Comment by XxCreepdawgxX** [comment]
Am I unable to open it because I'm still on Yosemite 10.10?

**Comment by mikeew83** [comment]
> Am I unable to open it because I'm still on Yosemite 10.10?

I don't think so, I am on Mojave latest build and have same issue.

**Comment by lazerwalker** [comment]
Yes, the current build appears to be missing ffmpeg, which likely means it won't work on any macOS version (or Windows, to see from some of the comments in here?)

**Comment by amydionne** [comment]
Process:               SideQuest [1579]
Path:                  /private/var/folders/*/SideQuest.app/Contents/MacOS/SideQuest
Identifier:            com.electron.sidequest
Version:               0.2.0 (0.2.0)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           SideQuest [1579]
User ID:               501

Date/Time:             2019-06-04 17:43:39.199 -0400
OS Version:            Mac OS X 10.14.5 (18F132)
Report Version:        12
Bridge OS Version:     3.5 (16P5125)
Anonymous UUID:        8A12D446-F6F9-62DB-F90A-265EA19D8369

Sleep/Wake UUID:       35B82492-799A-42C4-837B-FD8DBED1F6A8

Time Awake Since Boot: 5200 seconds
Time Since Wake:       2400 seconds

System Integrity Protection: enabled

Notes:                 Translocated Process

Crashed Thread:        0

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/libffmpeg.dylib
  Referenced from: /private/var/folders/q0/vr1807g56998nbyqdp3d4c6c0000gn/T/AppTranslocation/52FB8B27-9313-4723-8E6F-7B6B5A534D0B/d/SideQuest.app/Contents/Frameworks/Electron Framework.framework/Electron Framework
  Reason: image not found

Binary Images:
       0x10725b000 -        0x10725bffb +com.electron.sidequest (0.2.0 - 0.2.0) <6E272AEB-51ED-363B-B2D0-1E8B0C094EDF> /var/folders/*/SideQuest.app/Contents/MacOS/SideQuest
       0x10725e000 -        0x10cd7df47 +Electron Framework (0) <DE8BA951-A6F5-3B17-8ED5-0F3789BA1B8E> /var/folders/*/SideQuest.app/Contents/Frameworks/Electron Framework.framework/Electron Framework
       0x10d3a7000 -        0x10d3c2fff +Squirrel (0) <E4398068-33D3-3A00-9DBE-5ACC9B022501> /var/folders/*/SideQuest.app/Contents/Frameworks/Squirrel.framework/Squirrel
       0x10d3e4000 -        0x10d447ff7 +ReactiveCocoa (0) <701B20DE-3ADD-3643-B52A-E05744C30DB3> /var/folders/*/SideQuest.app/Contents/Frameworks/ReactiveCocoa.framework/ReactiveCocoa
       0x10d4bd000 -        0x10d4d1fff +Mantle (0) <31915DD6-48E6-3706-A076-C9D4CE17F4F6> /var/folders/*/SideQuest.app/Contents/Frameworks/Mantle.framework/Mantle
       0x112211000 -        0x11227b6ef  dyld (655.1.1) <CE635DB2-D47E-3C05-A0A3-6BD982E7E750> /usr/lib/dyld
    0x7fff3076b000 -     0x7fff30947ff3  com.apple.avfoundation (2.0 - 1548.22) <733E940A-7E18-3F0C-89AC-D7C6CA289BA6> /System/Library/Frameworks/AVFoundation.framework/Versions/A/AVFoundation
    0x7fff30b15000 -     0x7fff30b15fff  com.apple.Accelerate (1.11 - Accelerate 1.11) <762942CB-CFC9-3A0C-9645-A56523A06426> /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate
    0x7fff30b2d000 -     0x7fff311c6fef  com.apple.vImage (8.1 - ???) <EFFD9A9C-5563-3762-91CA-9F50FDB7A547> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vImage.framework/Versions/A/vImage
    0x7fff311c7000 -     0x7fff31440ff3  libBLAS.dylib (1243.200.4) <417CA0FC-B6CB-3FB3-ACBC-8914E3F62D20> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
    0x7fff31441000 -     0x7fff314b3ffb  libBNNS.dylib (38.250.1) <538D12A2-9B9D-3E22-9896-F90F6E69C06E> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBNNS.dylib
    0x7fff314b4000 -     0x7fff3185dff3  libLAPACK.dylib (1243.200.4) <92175DF4-863A-3780-909A-A3E5C410F2E9> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib
    0x7fff3185e000 -     0x7fff31873feb  libLinearAlgebra.dylib (1243.200.4) <CB671EE6-DEA1-391C-9B2B-AA09A46B4D7A> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLinearAlgebra.dylib
    0x7fff31874000 -     0x7fff31879ff3  libQuadrature.dylib (3.200.2) <1BAE7E22-2862-379F-B334-A3756067730F> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libQuadrature.dylib
    0x7fff3187a000 -     0x7fff318f6ff3  libSparse.dylib (79.200.5) <E78B33D3-672A-3C53-B512-D3DDB2E9AC8D> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparse.dylib
    0x7fff318f7000 -     0x7fff3190afe3  libSparseBLAS.dylib (1243.200.4) <E9243341-DB77-37C1-97C5-3DFA00DD70FA> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libSparseBLAS.dylib
    0x7fff3190b000 -     0x7fff31af2ff7  libvDSP.dylib (671.250.4) <7B110627-A9C1-3FB7-A077-0C7741BA25D8> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvDSP.dylib
    0x7fff31af3000 -     0x7fff31ba6ff7  libvMisc.dylib (671.250.4) <73637194-497A-39F9-90F5-004DA772AC69> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libvMisc.dylib
    0x7fff31ba7000 -     0x7fff31ba7fff  com.apple.Accelerate.vecLib (3.11 - vecLib 3.11) <74288115-EF61-30B6-843F-0593B31D4929> /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/vecLib
    0x7fff31c05000 -     0x7fff31d48fff  com.apple.AddressBook.framework (11.0 - 1894) <4D4DF3AE-C333-3646-8BA6-F0E8DFE1A46E> /System/Library/Frameworks/AddressBook.framework/Versions/A/AddressBook
    0x7fff31d49000 -     0x7fff32afeff3  com.apple.AppKit (6.9 - 1671.50.111) <933155DC-6404-3646-B6AE-087D34F611BE> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
    0x7fff32b50000 -     0x7fff32b50fff  com.apple.ApplicationServices (50.1 - 50.1) <4A6ACC13-2623-36B8-A196-2300980803CD> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices
    0x7fff32b51000 -     0x7fff32bbcfff  com.apple.ApplicationServices.ATS (377 - 453.11.2.2) <A258DA73-114B-3102-A056-4AAAD3CEB9DD> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/ATS
    0x7fff32c55000 -     0x7fff32d6cfff  libFontParser.dylib (228.6.2.3) <3602D55B-3B9E-3B3A-A814-08C1244A8AE4> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontParser.dylib
    0x7fff32d6d000 -     0x7fff32daffff  libFontRegistry.dylib (228.12.2.3) <2A56347B-2809-3407-A8B4-2AB88E484062> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ATS.framework/Versions/A/Resources/libFontRegistry.dylib
    0x7fff32ea0000 -     0x7fff32ea4ff3  com.apple.ColorSyncLegacy (4.13.0 - 1) <B53D557C-071F-3ED9-826A-847DEB465F62> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/ColorSyncLegacy.framework/Versions/A/ColorSyncLegacy
    0x7fff32f3f000 -     0x7fff32f91ff7  com.apple.HIServices (1.22 - 628) <26A8B7C0-41CC-3DB4-99D7-BD922866DC77> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/HIServices.framework/Versions/A/HIServices
    0x7fff32f92000 -     0x7fff32fa1fff  com.apple.LangAnalysis (1.7.0 - 1.7.0) <F5617A2A-FEA6-3832-B5BA-C2111B98786F> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/LangAnalysis.framework/Versions/A/LangAnalysis
    0x7fff32fa2000 -     0x7fff32febff7  com.apple.print.framework.PrintCore (14.2 - 503.8) <57C2FE32-0E74-3079-B626-C2D52F2D2717> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/PrintCore.framework/Versions/A/PrintCore
    0x7fff32fec000 -     0x7fff33025ff7  com.apple.QD (3.12 - 407.2) <28C7D39F-59C9-3314-BECC-67045487229C> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/QD.framework/Versions/A/QD
    0x7fff33026000 -     0x7fff33032fff  com.apple.speech.synthesis.framework (8.1.3 - 8.1.3) <80FA7439-EC77-3CEC-B5D1-5EFE9E503E11> /System/Library/Frameworks/ApplicationServices.framework/Versions/A/Frameworks/SpeechSynthesis.framework/Versions/A/SpeechSynthesis
    0x7fff33033000 -     0x7fff332aaff7  com.apple.audio.toolbox.AudioToolbox (1.14 - 1.14) <EB71583C-58C9-3CF8-92F9-2D57B24BBD4F> /System/Library/Frameworks/AudioToolbox.framework/Versions/A/AudioToolbox
    0x7fff332ac000 -     0x7fff332acfff  com.apple.audio.units.AudioUnit (1.14 - 1.14) <121176B6-7ABE-3AFB-A252-CCED694D98D4> /System/Library/Frameworks/AudioUnit.framework/Versions/A/AudioUnit
    0x7fff33604000 -     0x7fff339a5fff  com.apple.CFNetwork (978.0.7 - 978.0.7) <9FDEFE50-BE1B-386E-8B3C-AEF251069E52> /System/Library/Frameworks/CFNetwork.framework/Versions/A/CFNetwork
    0x7fff339ba000 -     0x7fff339bafff  com.apple.Carbon (158 - 158) <AC9E8C1E-2566-3C2C-8B6B-B5B4C9A1B8B7> /System/Library/Frameworks/Carbon.framework/Versions/A/Carbon
    0x7fff339bf000 -     0x7fff33cb5ff7  com.apple.HIToolbox (2.1.1 - 918.5) <F7DA4F20-123B-30FB-9C27-5AB1746DEF63> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
    0x7fff33cc0000 -     0x7fff33d55ff3  com.apple.ink.framework (10.9 - 225) <7C7E9483-2E91-3DD3-B1E0-C238F42CA0DD> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/Ink.framework/Versions/A/Ink
    0x7fff33d93000 -     0x7fff33d99ff7  com.apple.speech.recognition.framework (6.0.3 - 6.0.3) <3CC050FB-EBCB-3087-8EA5-F378C8F99217> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/SpeechRecognition.framework/Versions/A/SpeechRecognition
    0x7fff33ebb000 -     0x7fff33ebbfff  com.apple.Cocoa (6.11 - 23) <9378783B-65FE-3E78-A7D5-0D11EE9117AE> /System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa
    0x7fff33ec9000 -     0x7fff33f95ff7  com.apple.ColorSync (4.13.0 - 3340.7) <74B373AB-42C7-330D-A577-1C43BFA6024F> /System/Library/Frameworks/ColorSync.framework/Versions/A/ColorSync
    0x7fff34121000 -     0x7fff341a7fff  com.apple.audio.CoreAudio (4.3.0 - 4.3.0) <1E8E64E6-0E58-375A-97F7-07CB4EE181AC> /System/Library/Frameworks/CoreAudio.framework/Versions/A/CoreAudio
    0x7fff3420b000 -     0x7fff34235ffb  com.apple.CoreBluetooth (1.0 - 1) <F08FE8C0-DD18-37E0-B707-3EF704A73FD6> /System/Library/Frameworks/CoreBluetooth.framework/Versions/A/CoreBluetooth
    0x7fff34236000 -     0x7fff345bafe3  com.apple.CoreData (120 - 866.5) <37F35F2E-FE57-3E3F-A1E5-AD274A8C1FDE> /System/Library/Frameworks/CoreData.framework/Versions/A/CoreData
    0x7fff345bb000 -     0x7fff346a2ff7  com.apple.CoreDisplay (101.3 - 109.19) <B00A19C2-4A6A-388C-8AD6-33A42D10D7F0> /System/Library/Frameworks/CoreDisplay.framework/Versions/A/CoreDisplay
    0x7fff346a3000 -     0x7fff34ae7fff  com.apple.CoreFoundation (6.9 - 1575.12) <64C38BF2-1869-33C8-B1B6-DCA9C824C0E4> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
    0x7fff34ae9000 -     0x7fff35178fe7  com.apple.CoreGraphics (2.0 - 1260.2) <23436C9A-7D62-3E9F-867A-798C93B52080> /System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics
    0x7fff3517a000 -     0x7fff3549afff  com.apple.CoreImage (14.4.0 - 750.0.140) <96FA4D1B-AC88-305C-8F4D-D7D44657B733> /System/Library/Frameworks/CoreImage.framework/Versions/A/CoreImage
    0x7fff35514000 -     0x7fff3556aff7  com.apple.audio.midi.CoreMIDI (1.10 - 88) <B5531AC1-FEDF-331C-AE38-883809BEF972> /System/Library/Frameworks/CoreMIDI.framework/Versions/A/CoreMIDI
    0x7fff35797000 -     0x7fff3589bff7  com.apple.CoreMedia (1.0 - 2288.40) <9BB054FB-68F1-3F03-A8B3-3E55471A6F9B> /System/Library/Frameworks/CoreMedia.framework/Versions/A/CoreMedia
    0x7fff358f8000 -     0x7fff358f8fff  com.apple.CoreServices (945 - 945) <E937BA69-19D5-3A30-B31E-8A6B94E9D9A0> /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
    0x7fff358f9000 -     0x7fff35975ff7  com.apple.AE (773 - 773) <55AE7C9E-27C3-30E9-A047-3B92A6FD53B4> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/AE.framework/Versions/A/AE
    0x7fff35976000 -     0x7fff35c4dfff  com.apple.CoreServices.CarbonCore (1178.32 - 1178.32) <52F5484E-94B1-37B7-B1AC-E210DF5721AE> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/CarbonCore.framework/Versions/A/CarbonCore
    0x7fff35c4e000 -     0x7fff35c96ff7  com.apple.DictionaryServices (1.2 - 284.16.4) <C3D15FD1-759B-3815-8128-42D8220BC459> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/DictionaryServices.framework/Versions/A/DictionaryServices
    0x7fff35c97000 -     0x7fff35c9fffb  com.apple.CoreServices.FSEvents (1239.200.12 - 1239.200.12) <8406D379-8D33-3611-861B-7ABD26DB50D2> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/FSEvents.framework/Versions/A/FSEvents
    0x7fff35ca0000 -     0x7fff35e52fff  com.apple.LaunchServices (945 - 945) <05FA145B-75AA-3AE6-BB82-F156B29F4FF1> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/LaunchServices.framework/Versions/A/LaunchServices
    0x7fff35e53000 -     0x7fff35ef1ff7  com.apple.Metadata (10.7.0 - 1191.57) <BFFAED00-2560-318A-BB8F-4E7E5123EC61> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/Metadata.framework/Versions/A/Metadata
    0x7fff35ef2000 -     0x7fff35f3cff7  com.apple.CoreServices.OSServices (945 - 945) <119C5548-59C3-3F94-B23F-8AF02CA10EAA> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/OSServices.framework/Versions/A/OSServices
    0x7fff35f3d000 -     0x7fff35fa4ff7  com.apple.SearchKit (1.4.0 - 1.4.0) <DA08AA6F-A6F1-36C0-87F4-E26294E51A3A> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SearchKit.framework/Versions/A/SearchKit
    0x7fff35fa5000 -     0x7fff35fc6ff3  com.apple.coreservices.SharedFileList (71.28 - 71.28) <487A8464-729E-305A-B5D1-E3FE8EB9CFC5> /System/Library/Frameworks/CoreServices.framework/Versions/A/Frameworks/SharedFileList.framework/Versions/A/SharedFileList
    0x7fff362d1000 -     0x7fff36433ff3  com.apple.CoreText (352.0 - 584.26.3.2) <A99FD623-7F6E-3C77-8241-E98F664EC039> /System/Library/Frameworks/CoreText.framework/Versions/A/CoreText
    0x7fff36434000 -     0x7fff36473fff  com.apple.CoreVideo (1.8 - 0.0) <1AD23C58-E68E-329B-A1FA-6BF7DD577687> /System/Library/Frameworks/CoreVideo.framework/Versions/A/CoreVideo
    0x7fff36474000 -     0x7fff36503fff  com.apple.framework.CoreWLAN (13.0 - 1375.2) <F2EA31D8-53BF-32E8-B537-64B4F2BF4B6E> /System/Library/Frameworks/CoreWLAN.framework/Versions/A/CoreWLAN
    0x7fff3675a000 -     0x7fff3675fffb  com.apple.DiskArbitration (2.7 - 2.7) <F481F2C0-884E-3265-8111-ABBEC93F0920> /System/Library/Frameworks/DiskArbitration.framework/Versions/A/DiskArbitration
    0x7fff36922000 -     0x7fff36924ff3  com.apple.ForceFeedback (1.0.6 - 1.0.6) <205B0C3E-4E8D-3347-A993-469F06ABBFF9> /System/Library/Frameworks/ForceFeedback.framework/Versions/A/ForceFeedback
    0x7fff36925000 -     0x7fff36cd2fff  com.apple.Foundation (6.9 - 1575.12) <AD258ED7-64AE-367C-AE5F-48D45089D9F2> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
    0x7fff36d41000 -     0x7fff36d70ffb  com.apple.GSS (4.0 - 2.0) <E41430E5-713B-3E53-BF7E-A3991A881B62> /System/Library/Frameworks/GSS.framework/Versions/A/GSS
    0x7fff36d71000 -     0x7fff36d8aff3  com.apple.GameController (1.0 - 1) <9339D37B-54EA-31C5-A4A5-2135288BA992> /System/Library/Frameworks/GameController.framework/Versions/A/GameController
    0x7fff36e70000 -     0x7fff36f7afff  com.apple.Bluetooth (6.0.12 - 6.0.12f1) <58A607F0-A818-3535-A7BD-858D5BB0A807> /System/Library/Frameworks/IOBluetooth.framework/Versions/A/IOBluetooth
    0x7fff36fdc000 -     0x7fff3706bfff  com.apple.framework.IOKit (2.0.2 - 1483.260.4) <CCA92450-8808-37EC-B68E-2C2846B086CE> /System/Library/Frameworks/IOKit.framework/Versions/A/IOKit
    0x7fff3706d000 -     0x7fff3707cffb  com.apple.IOSurface (255.5.4 - 255.5.4) <83F58E47-0D06-3E78-9D19-D38D3205464B> /System/Library/Frameworks/IOSurface.framework/Versions/A/IOSurface
    0x7fff370d0000 -     0x7fff3725bfef  com.apple.ImageIO.framework (3.3.0 - 1850.2) <E1CC52A4-1074-3997-849B-42EF20EE1ADC> /System/Library/Frameworks/ImageIO.framework/Versions/A/ImageIO
    0x7fff3725c000 -     0x7fff37260ffb  libGIF.dylib (1850.2) <FB52622F-2A7D-36D0-B821-35DB2BAF9EE8> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libGIF.dylib
    0x7fff37261000 -     0x7fff3733dfef  libJP2.dylib (1850.2) <7FAC16D8-1144-30A1-BABC-008423C7A851> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJP2.dylib
    0x7fff3733e000 -     0x7fff37363feb  libJPEG.dylib (1850.2) <78401F6E-345B-3179-A82A-01F813F15D95> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libJPEG.dylib
    0x7fff37626000 -     0x7fff3764cfeb  libPng.dylib (1850.2) <FF3ABAE1-43F0-3A76-A284-398397693D46> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libPng.dylib
    0x7fff3764d000 -     0x7fff3764fffb  libRadiance.dylib (1850.2) <92529333-0935-341B-B271-69C235FEF6B5> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libRadiance.dylib
    0x7fff37650000 -     0x7fff3769dfe7  libTIFF.dylib (1850.2) <FFC74F3B-16F9-38FE-9D3F-D8DC6DEDB9E9> /System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/libTIFF.dylib
    0x7fff38814000 -     0x7fff3882dfff  com.apple.Kerberos (3.0 - 1) <DB1E0679-37E1-3B93-9789-32F63D660C3B> /System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos
    0x7fff388cf000 -     0x7fff388e3fff  com.apple.LocalAuthentication (1.0 - 425.250.11) <80CEEB4D-3FAF-3BDB-B25C-A684888D9672> /System/Library/Frameworks/LocalAuthentication.framework/Versions/A/LocalAuthentication
    0x7fff39249000 -     0x7fff392f1ff7  com.apple.Metal (161.9.16 - 161.9.16) <A93AA7FC-B72A-3CBF-9932-81DDDFA1DFCD> /System/Library/Frameworks/Metal.framework/Versions/A/Metal
    0x7fff3930d000 -     0x7fff3932cff7  com.apple.MetalPerformanceShaders.MPSCore (1.0 - 1) <EE85B6D5-5C12-3CE8-8C33-54A203EDC76E> /System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSCore.framework/Versions/A/MPSCore
    0x7fff3932d000 -     0x7fff393a9fe7  com.apple.MetalPerformanceShaders.MPSImage (1.0 - 1) <5A78F2DC-9D2B-37BA-B21F-ECFFCC6F7720> /System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSImage.framework/Versions/A/MPSImage
    0x7fff393aa000 -     0x7fff393d1fff  com.apple.MetalPerformanceShaders.MPSMatrix (1.0 - 1) <096DB3E6-7DDF-3B47-857A-46A203642ACF> /System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSMatrix.framework/Versions/A/MPSMatrix
    0x7fff393d2000 -     0x7fff394fdff7  com.apple.MetalPerformanceShaders.MPSNeuralNetwork (1.0 - 1) <D2128338-F1E8-3BEE-A729-AC13A524DD78> /System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSNeuralNetwork.framework/Versions/A/MPSNeuralNetwork
    0x7fff394fe000 -     0x7fff39518fff  com.apple.MetalPerformanceShaders.MPSRayIntersector (1.0 - 1) <ADA5A93C-F79E-3931-840D-48C6B6B305EA> /System/Library/Frameworks/MetalPerformanceShaders.framework/Frameworks/MPSRayIntersector.framework/Versions/A/MPSRayIntersector
    0x7fff39519000 -     0x7fff3951aff7  com.apple.MetalPerformanceShaders.MetalPerformanceShaders (1.0 - 1) <515A92B3-A23B-3AC2-8D1B-85B248595B76> /System/Library/Frameworks/MetalPerformanceShaders.framework/Versions/A/MetalPerformanceShaders
    0x7fff3a311000 -     0x7fff3a31dff7  com.apple.NetFS (6.0 - 4.0) <E917806F-0607-3292-B2D6-A15404D61B99> /System/Library/Frameworks/NetFS.framework/Versions/A/NetFS
    0x7fff3cdbb000 -     0x7fff3ce12ff7  com.apple.opencl (2.15.3 - 2.15.3) <AA9CE8F7-DB7C-35A5-A019-2207D1BBFF38> /System/Library/Frameworks/OpenCL.framework/Versions/A/OpenCL
    0x7fff3ce13000 -     0x7fff3ce2eff7  com.apple.CFOpenDirectory (10.14 - 207.200.4) <F03D84EB-49B2-3A00-9127-B9A269824026> /System/Library/Frameworks/OpenDirectory.framework/Versions/A/Frameworks/CFOpenDirectory.framework/Versions/A/CFOpenDirectory
    0x7fff3ce2f000 -     0x7fff3ce3affb  com.apple.OpenDirectory (10.14 - 207.200.4) <A8020CEE-5B78-3581-A735-EA2833683F31> /System/Library/Frameworks/OpenDirectory.framework/Versions/A/OpenDirectory
    0x7fff3d78a000 -     0x7fff3d78cfff  libCVMSPluginSupport.dylib (17.7.2) <247121BB-626C-318E-B24E-3D436E752DF7> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCVMSPluginSupport.dylib
    0x7fff3d78d000 -     0x7fff3d792ff3  libCoreFSCache.dylib (166.2) <222C2A4F-7E32-30F6-8459-2FAB98073A3D> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCoreFSCache.dylib
    0x7fff3d793000 -     0x7fff3d797fff  libCoreVMClient.dylib (166.2) <6789ECD4-91DD-32EF-A1FD-F27D2344CD8B> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libCoreVMClient.dylib
    0x7fff3d798000 -     0x7fff3d7a0ff7  libGFXShared.dylib (17.7.2) <B75A6E93-E5BE-32A8-8D46-00D8E909DFD3> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGFXShared.dylib
    0x7fff3d7a1000 -     0x7fff3d7acfff  libGL.dylib (17.7.2) <E88FF023-7E15-3E95-88FD-6C415550F769> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGL.dylib
    0x7fff3d7ad000 -     0x7fff3d7e7fe7  libGLImage.dylib (17.7.2) <B4C140F3-AF8C-30C8-A03E-B16A20BB05A7> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLImage.dylib
    0x7fff3d95b000 -     0x7fff3d999fff  libGLU.dylib (17.7.2) <EA69FA36-3F20-337C-A6B3-EC49DC2F12F5> /System/Library/Frameworks/OpenGL.framework/Versions/A/Libraries/libGLU.dylib
    0x7fff3e336000 -     0x7fff3e345ffb  com.apple.opengl (17.7.2 - 17.7.2) <8EFFA6F9-ABE8-3131-9D3F-37E9796DDF34> /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
    0x7fff3f147000 -     0x7fff3f147fff  com.apple.quartzframework (1.5 - 23) <62B9AB40-3BE3-3A81-B132-3168898BD748> /System/Library/Frameworks/Quartz.framework/Versions/A/Quartz
    0x7fff3f148000 -     0x7fff3f39efff  com.apple.QuartzCore (1.11 - 699.27.4.1) <BB3BBD58-84DF-3849-B4ED-5FF8DA4015B8> /System/Library/Frameworks/QuartzCore.framework/Versions/A/QuartzCore
    0x7fff3fbd3000 -     0x7fff3fed2fff  com.apple.security (7.0 - 58286.260.20) <59B4100E-5A31-3EF4-954A-2A69341995F3> /System/Library/Frameworks/Security.framework/Versions/A/Security
    0x7fff3fed3000 -     0x7fff3ff5ffff  com.apple.securityfoundation (6.0 - 55185.260.1) <DB8EB672-5D3A-3082-A62E-4D75C81D94A4> /System/Library/Frameworks/SecurityFoundation.framework/Versions/A/SecurityFoundation
    0x7fff3ff60000 -     0x7fff3ff90ffb  com.apple.securityinterface (10.0 - 55109.200.8) <4F6896A5-4947-39A9-817A-3C23D97946AD> /System/Library/Frameworks/SecurityInterface.framework/Versions/A/SecurityInterface
    0x7fff3ff91000 -     0x7fff3ff95fff  com.apple.xpc.ServiceManagement (1.0 - 1) <4AD0F8EE-884C-3FA7-85D9-27F201BC3C64> /System/Library/Frameworks/ServiceManagement.framework/Versions/A/ServiceManagement
    0x7fff401e3000 -     0x7fff401f9ffb  com.apple.StoreKit (1.0 - 1) <5E26D23D-D85C-32EF-AE44-2B0437893274> /System/Library/Frameworks/StoreKit.framework/Versions/A/StoreKit
    0x7fff4032d000 -     0x7fff4039afff  com.apple.SystemConfiguration (1.17 - 1.17) <1562078D-3251-31B2-B9C6-2667C567F883> /System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration
    0x7fff405f9000 -     0x7fff40958ff7  com.apple.VideoToolbox (1.0 - 2288.40) <17BD9E7F-3B8E-3B62-BACE-AE04824F4B7B> /System/Library/Frameworks/VideoToolbox.framework/Versions/A/VideoToolbox
    0x7fff43619000 -     0x7fff436befeb  com.apple.APFS (1.0 - 1) <C001DCC2-32BE-327A-AE03-281885BD93BF> /System/Library/PrivateFrameworks/APFS.framework/Versions/A/APFS
    0x7fff440d0000 -     0x7fff440d1ff7  com.apple.AggregateDictionary (1.0 - 1) <A6AF8AC4-1F25-37C4-9157-A02E9C200926> /System/Library/PrivateFrameworks/AggregateDictionary.framework/Versions/A/AggregateDictionary
    0x7fff446cf000 -     0x7fff446fbff7  com.apple.framework.Apple80211 (13.0 - 1380.2) <3850E3A9-E00A-374E-87EA-AB01E503E769> /System/Library/PrivateFrameworks/Apple80211.framework/Versions/A/Apple80211
    0x7fff44823000 -     0x7fff44832fc7  com.apple.AppleFSCompression (96.200.3 - 1.0) <3CF60CE8-976E-3CB8-959D-DD0948C1C2DE> /System/Library/PrivateFrameworks/AppleFSCompression.framework/Versions/A/AppleFSCompression
    0x7fff4492e000 -     0x7fff44939fff  com.apple.AppleIDAuthSupport (1.0 - 1) <2E9D1398-DBE6-328B-ADDA-20FA5FAD7405> /System/Library/PrivateFrameworks/AppleIDAuthSupport.framework/Versions/A/AppleIDAuthSupport
    0x7fff4497a000 -     0x7fff449c3ff3  com.apple.AppleJPEG (1.0 - 1) <4C1F426B-7D77-3980-9633-7DBD8C666B9A> /System/Library/PrivateFrameworks/AppleJPEG.framework/Versions/A/AppleJPEG
    0x7fff44c17000 -     0x7fff44c39fff  com.apple.applesauce (1.0 - ???) <F49107C7-3C51-3024-8EF1-C57643BE4F3B> /System/Library/PrivateFrameworks/AppleSauce.framework/Versions/A/AppleSauce
    0x7fff44d98000 -     0x7fff44dacffb  com.apple.AssertionServices (1.0 - 1) <11FC4241-6069-3B3A-9BCE-9B68F3A4D2AB> /System/Library/PrivateFrameworks/AssertionServices.framework/Versions/A/AssertionServices
    0x7fff4517b000 -     0x7fff45267ff7  com.apple.AuthKit (1.0 - 1) <34496FD5-4BAB-3DF3-BF7F-70F38C66848D> /System/Library/PrivateFrameworks/AuthKit.framework/Versions/A/AuthKit
    0x7fff45429000 -     0x7fff45431fff  com.apple.coreservices.BackgroundTaskManagement (1.0 - 57.1) <2A396FC0-7B79-3088-9A82-FB93C1181A57> /System/Library/PrivateFrameworks/BackgroundTaskManagement.framework/Versions/A/BackgroundTaskManagement
    0x7fff45432000 -     0x7fff454c7fff  com.apple.backup.framework (1.10.5 - ???) <BAFC1D7F-9D75-3C45-AAD4-569D13E445C5> /System/Library/PrivateFrameworks/Backup.framework/Versions/A/Backup
    0x7fff454c8000 -     0x7fff45535ff3  com.apple.BaseBoard (360.28 - 360.28) <DBE27FB4-3FFD-3E9A-846C-F6E7D797286A> /System/Library/PrivateFrameworks/BaseBoard.framework/Versions/A/BaseBoard
    0x7fff4719f000 -     0x7fff471a8ffb  com.apple.CommonAuth (4.0 - 2.0) <81C987FD-69EB-344B-94D9-B9D93624D257> /System/Library/PrivateFrameworks/CommonAuth.framework/Versions/A/CommonAuth
    0x7fff47e80000 -     0x7fff47e91ff7  com.apple.CoreEmoji (1.0 - 69.19.9) <228457B3-E191-356E-9A5B-3C0438D05FBA> /System/Library/PrivateFrameworks/CoreEmoji.framework/Versions/A/CoreEmoji
    0x7fff4843a000 -     0x7fff484a0ff7  com.apple.CoreNLP (1.0 - 130.15.22) <27877820-17D0-3B02-8557-4014E876CCC7> /System/Library/PrivateFrameworks/CoreNLP.framework/Versions/A/CoreNLP
    0x7fff4874d000 -     0x7fff48755ff7  com.apple.CorePhoneNumbers (1.0 - 1) <7A826DA2-4725-39D8-AF27-6F034DC53C4E> /System/Library/PrivateFrameworks/CorePhoneNumbers.framework/Versions/A/CorePhoneNumbers
    0x7fff48cc9000 -     0x7fff48d4dfff  com.apple.CoreSymbolication (10.2 - 64490.25.1) <28B2FF2D-3FDE-3A20-B343-341E5BD4E22F> /System/Library/PrivateFrameworks/CoreSymbolication.framework/Versions/A/CoreSymbolication
    0x7fff48ddd000 -     0x7fff48f08ff7  com.apple.coreui (2.1 - 499.10) <105F61F3-D0AE-332B-A485-8615187BB409> /System/Library/PrivateFrameworks/CoreUI.framework/Versions/A/CoreUI
    0x7fff48f09000 -     0x7fff490a7fff  com.apple.CoreUtils (5.8 - 580.19) <CF9C5677-4946-3328-8CE3-C70F2E8599D8> /System/Library/PrivateFrameworks/CoreUtils.framework/Versions/A/CoreUtils
    0x7fff490fb000 -     0x7fff4915eff7  com.apple.framework.CoreWiFi (13.0 - 1375.2) <77E078A7-A34F-333B-B50F-8829537D6E9D> /System/Library/PrivateFrameworks/CoreWiFi.framework/Versions/A/CoreWiFi
    0x7fff4915f000 -     0x7fff49170ff7  com.apple.CrashReporterSupport (10.13 - 938.25) <C7665EC7-9389-3717-B152-A59D7872C63C> /System/Library/PrivateFrameworks/CrashReporterSupport.framework/Versions/A/CrashReporterSupport
    0x7fff49200000 -     0x7fff4920ffff  com.apple.framework.DFRFoundation (1.0 - 211.1) <E3F02F2A-2059-39CC-85DA-969676EB88EB> /System/Library/PrivateFrameworks/DFRFoundation.framework/Versions/A/DFRFoundation
    0x7fff49210000 -     0x7fff49214fff  com.apple.DSExternalDisplay (3.1 - 380) <645C719D-05D4-3CD7-84D7-4C5218859E0C> /System/Library/PrivateFrameworks/DSExternalDisplay.framework/Versions/A/DSExternalDisplay
    0x7fff49295000 -     0x7fff4930aff3  com.apple.datadetectorscore (7.0 - 590.24) <72F0CBF1-D3E9-36C5-B877-F42E1EC59A0E> /System/Library/PrivateFrameworks/DataDetectorsCore.framework/Versions/A/DataDetectorsCore
    0x7fff49356000 -     0x7fff49393ff7  com.apple.DebugSymbols (190 - 190) <6F4FAACA-E06B-38AD-A0C2-14EA5408A231> /System/Library/PrivateFrameworks/DebugSymbols.framework/Versions/A/DebugSymbols
    0x7fff49394000 -     0x7fff494cfff7  com.apple.desktopservices (1.13.5 - ???) <4AF2E011-2CB0-3588-83CB-882ADDA78CAD> /System/Library/PrivateFrameworks/DesktopServicesPriv.framework/Versions/A/DesktopServicesPriv
    0x7fff4a40c000 -     0x7fff4a827fff  com.apple.vision.FaceCore (3.3.4 - 3.3.4) <A576E2DA-BF6F-3B18-8FEB-324E5C5FA9BD> /System/Library/PrivateFrameworks/FaceCore.framework/Versions/A/FaceCore
    0x7fff4f77c000 -     0x7fff4f781fff  com.apple.GPUWrangler (3.40.16 - 3.40.16) <B9D4F2DD-5D11-32FF-B5BE-29E137458728> /System/Library/PrivateFrameworks/GPUWrangler.framework/Versions/A/GPUWrangler
    0x7fff5058c000 -     0x7fff5059bfff  com.apple.GraphVisualizer (1.0 - 5) <48D020B7-5938-3FAE-B468-E291AEE2C06F> /System/Library/PrivateFrameworks/GraphVisualizer.framework/Versions/A/GraphVisualizer
    0x7fff506eb000 -     0x7fff5075fffb  com.apple.Heimdal (4.0 - 2.0) <5E8F6B79-960D-3030-B889-D29A4015D205> /System/Library/PrivateFrameworks/Heimdal.framework/Versions/A/Heimdal
    0x7fff51a4e000 -     0x7fff51a55ffb  com.apple.IOAccelerator (404.9 - 404.9) <F9C604B3-3DF7-3EE7-8A44-7877FAF2E6A4> /System/Library/PrivateFrameworks/IOAccelerator.framework/Versions/A/IOAccelerator
    0x7fff51a59000 -     0x7fff51a71fff  com.apple.IOPresentment (1.0 - 42.6) <55CB91EF-90F1-3D0A-A59E-2F75CD2EB4AE> /System/Library/PrivateFrameworks/IOPresentment.framework/Versions/A/IOPresentment
    0x7fff51e19000 -     0x7fff51e46ff7  com.apple.IconServices (379 - 379) <7BAD562D-4FA3-3E11-863C-1EEBE2406D2C> /System/Library/PrivateFrameworks/IconServices.framework/Versions/A/IconServices
    0x7fff520da000 -     0x7fff520ecff3  com.apple.security.KeychainCircle.KeychainCircle (1.0 - 1) <5AAC527E-D197-39B2-A6C9-E0D0FB9CA9B9> /System/Library/PrivateFrameworks/KeychainCircle.framework/Versions/A/KeychainCircle
    0x7fff52107000 -     0x7fff521e2ff7  com.apple.LanguageModeling (1.0 - 159.15.15) <3DE3CE61-542B-37B7-883E-4B9717CAC65F> /System/Library/PrivateFrameworks/LanguageModeling.framework/Versions/A/LanguageModeling
    0x7fff521e3000 -     0x7fff5221fff7  com.apple.Lexicon-framework (1.0 - 33.15.10) <4B5E843E-2809-3E70-9560-9254E2656419> /System/Library/PrivateFrameworks/Lexicon.framework/Versions/A/Lexicon
    0x7fff52226000 -     0x7fff5222bfff  com.apple.LinguisticData (1.0 - 238.25) <61910887-D5CE-3567-9A5F-6DFEC942EC1F> /System/Library/PrivateFrameworks/LinguisticData.framework/Versions/A/LinguisticData
    0x7fff52f20000 -     0x7fff52f48ff7  com.apple.spotlight.metadata.utilities (1.0 - 1191.57) <38BB1FB7-3336-384C-B71F-4D0D402EB606> /System/Library/PrivateFrameworks/MetadataUtilities.framework/Versions/A/MetadataUtilities
    0x7fff52f49000 -     0x7fff52fd6ff7  com.apple.gpusw.MetalTools (1.0 - 1) <DBC4718E-F588-382D-A415-405D0028804D> /System/Library/PrivateFrameworks/MetalTools.framework/Versions/A/MetalTools
    0x7fff53183000 -     0x7fff5319effb  com.apple.MobileKeyBag (2.0 - 1.0) <4678FECE-538B-381C-80F2-27AA4805F160> /System/Library/PrivateFrameworks/MobileKeyBag.framework/Versions/A/MobileKeyBag
    0x7fff53227000 -     0x7fff53251ffb  com.apple.MultitouchSupport.framework (2450.1 - 2450.1) <3BF81F33-89A8-3A2D-84B3-B343020B6D98> /System/Library/PrivateFrameworks/MultitouchSupport.framework/Versions/A/MultitouchSupport
    0x7fff5348d000 -     0x7fff53497fff  com.apple.NetAuth (6.2 - 6.2) <0D01BBE5-0269-310D-B148-D19DAE143DEB> /System/Library/PrivateFrameworks/NetAuth.framework/Versions/A/NetAuth
    0x7fff53cf8000 -     0x7fff53d49ff3  com.apple.OTSVG (1.0 - ???) <3976F8CC-1CFB-3E09-9009-FE985550565B> /System/Library/PrivateFrameworks/OTSVG.framework/Versions/A/OTSVG
    0x7fff54ece000 -     0x7fff54eddff7  com.apple.PerformanceAnalysis (1.218.2 - 218.2) <65F3DB3E-6D4E-33A0-B510-EF768D323DAB> /System/Library/PrivateFrameworks/PerformanceAnalysis.framework/Versions/A/PerformanceAnalysis
    0x7fff56d6d000 -     0x7fff56d8bff7  com.apple.ProtocolBuffer (1 - 263.2) <907D6C95-D050-31DE-99CA-16A5135BC6F9> /System/Library/PrivateFrameworks/ProtocolBuffer.framework/Versions/A/ProtocolBuffer
    0x7fff56f23000 -     0x7fff56f73fff  com.apple.ROCKit (27.6 - 27.6) <756C2253-E8B1-3C48-9945-DE8D6AD24DE2> /System/Library/PrivateFrameworks/ROCKit.framework/Versions/A/ROCKit
    0x7fff570c2000 -     0x7fff570e4fff  com.apple.RemoteViewServices (2.0 - 128) <8FB0E4EB-DCBB-32E6-94C6-AA9BA9EE4CAC> /System/Library/PrivateFrameworks/RemoteViewServices.framework/Versions/A/RemoteViewServices
    0x7fff588e9000 -     0x7fff58a07fff  com.apple.Sharing (1288.43 - 1288.43) <B9AB8C05-F2F2-371F-A9AF-1D139E2CE54C> /System/Library/PrivateFrameworks/Sharing.framework/Versions/A/Sharing
    0x7fff59818000 -     0x7fff59ac6ffb  com.apple.SkyLight (1.600.0 - 340.26) <CD2A7069-6CEA-3737-9E9D-7AE2FED912AB> /System/Library/PrivateFrameworks/SkyLight.framework/Versions/A/SkyLight
    0x7fff5a268000 -     0x7fff5a274fff  com.apple.SpeechRecognitionCore (5.0.21 - 5.0.21) <7A6A67DB-C813-328E-AAFB-D267A5B50B3D> /System/Library/PrivateFrameworks/SpeechRecognitionCore.framework/Versions/A/SpeechRecognitionCore
    0x7fff5a9c5000 -     0x7fff5aa50fc7  com.apple.Symbolication (10.2 - 64490.38.1) <9FDCC98D-5B32-35AD-A9BF-94DF2B78507F> /System/Library/PrivateFrameworks/Symbolication.framework/Versions/A/Symbolication
    0x7fff5af38000 -     0x7fff5af44ffb  com.apple.TCC (1.0 - 1) <968ECAE1-FBB9-3478-BDCE-85E54B6C422D> /System/Library/PrivateFrameworks/TCC.framework/Versions/A/TCC
    0x7fff5b1aa000 -     0x7fff5b272ff3  com.apple.TextureIO (3.8.4 - 3.8.1) <7CEAC05A-D283-3D5A-B1E3-C849285FA0BF> /System/Library/PrivateFrameworks/TextureIO.framework/Versions/A/TextureIO
    0x7fff5b32f000 -     0x7fff5b4e5ff7  com.apple.UIFoundation (1.0 - 551) <E9D17DAD-230D-390A-B117-3D100C5AF5F2> /System/Library/PrivateFrameworks/UIFoundation.framework/Versions/A/UIFoundation
    0x7fff5ca11000 -     0x7fff5ca14fff  com.apple.dt.XCTTargetBootstrap (1.0 - 14490.66) <7AE3457F-AF40-3508-93FB-1D9E31EB1C9D> /System/Library/PrivateFrameworks/XCTTargetBootstrap.framework/Versions/A/XCTTargetBootstrap
    0x7fff5ce15000 -     0x7fff5ce17ffb  com.apple.loginsupport (1.0 - 1) <3F8D6334-BCD6-36C1-BA20-CC8503A84375> /System/Library/PrivateFrameworks/login.framework/Versions/A/Frameworks/loginsupport.framework/Versions/A/loginsupport
    0x7fff5d0e1000 -     0x7fff5d115fff  libCRFSuite.dylib (41.15.4) <406DAC06-0C77-3F90-878B-4D38F11F0256> /usr/lib/libCRFSuite.dylib
    0x7fff5d118000 -     0x7fff5d122ff7  libChineseTokenizer.dylib (28.15.3) <9B7F6109-3A5D-3641-9A7E-31D2239D73EE> /usr/lib/libChineseTokenizer.dylib
    0x7fff5d1b0000 -     0x7fff5d1b1ffb  libDiagnosticMessagesClient.dylib (107) <A14D0819-0970-34CD-8680-80E4D7FE8C2C> /usr/lib/libDiagnosticMessagesClient.dylib
    0x7fff5d1e8000 -     0x7fff5d43fff3  libFosl_dynamic.dylib (18.3.4) <45E50221-1A39-3AB3-8267-C1E4E39ECF83> /usr/lib/libFosl_dynamic.dylib
    0x7fff5d490000 -     0x7fff5d4aefff  libMobileGestalt.dylib (645.260.6) <0232958E-1EF2-3627-B8FF-09C6522594C7> /usr/lib/libMobileGestalt.dylib
    0x7fff5d4af000 -     0x7fff5d4affff  libOpenScriptingUtil.dylib (179.1) <4D603146-EDA5-3A74-9FF8-4F75D8BB9BC6> /usr/lib/libOpenScriptingUtil.dylib
    0x7fff5d5ef000 -     0x7fff5d5f0ffb  libSystem.B.dylib (1252.250.1) <1A13E822-B59C-3A36-A2E4-9968149714F9> /usr/lib/libSystem.B.dylib
    0x7fff5d66c000 -     0x7fff5d66dfff  libThaiTokenizer.dylib (2.15.1) <ADB37DC3-7D9B-3E73-A72A-BCC3433C937A> /usr/lib/libThaiTokenizer.dylib
    0x7fff5d67f000 -     0x7fff5d695ffb  libapple_nghttp2.dylib (1.24.1) <6F04250A-6686-3FDC-9A8D-290C64B06502> /usr/lib/libapple_nghttp2.dylib
    0x7fff5d696000 -     0x7fff5d6bfffb  libarchive.2.dylib (54.250.1) <47289946-8504-3966-9127-6CE39993DC2C> /usr/lib/libarchive.2.dylib
    0x7fff5d6c0000 -     0x7fff5d73ffff  libate.dylib (1.13.8) <92B44EDB-369D-3EE8-AEC5-61F8B9313DBF> /usr/lib/libate.dylib
    0x7fff5d743000 -     0x7fff5d743ff3  libauto.dylib (187) <3E3780E1-96F3-3A22-91C5-92F9A5805518> /usr/lib/libauto.dylib
    0x7fff5d813000 -     0x7fff5d823ffb  libbsm.0.dylib (39.200.18) <CF381E0B-025B-364F-A83D-2527E03F1AA3> /usr/lib/libbsm.0.dylib
    0x7fff5d824000 -     0x7fff5d831fff  libbz2.1.0.dylib (38.200.3) <272953A1-8D36-329B-BDDB-E887B347710F> /usr/lib/libbz2.1.0.dylib
    0x7fff5d832000 -     0x7fff5d885ff7  libc++.1.dylib (400.9.4) <9A60A190-6C34-339F-BB3D-AACE942009A4> /usr/lib/libc++.1.dylib
    0x7fff5d886000 -     0x7fff5d89bff7  libc++abi.dylib (400.17) <38C09CED-9090-3719-90F3-04A2749F5428> /usr/lib/libc++abi.dylib
    0x7fff5d89c000 -     0x7fff5d89cff3  libcharset.1.dylib (51.200.6) <2A27E064-314C-359C-93FC-8A9B06206174> /usr/lib/libcharset.1.dylib
    0x7fff5d89d000 -     0x7fff5d8adffb  libcmph.dylib (6.15.1) <9C52B2FE-179F-32AC-B87E-2AFC49ABF817> /usr/lib/libcmph.dylib
    0x7fff5d8ae000 -     0x7fff5d8c6ffb  libcompression.dylib (52.250.2) <7F4BB18C-1FB4-3825-8D8B-6E6B168774C6> /usr/lib/libcompression.dylib
    0x7fff5db3b000 -     0x7fff5db51fff  libcoretls.dylib (155.220.1) <4C64BE3E-41E3-3020-8BB7-07E90C0C861C> /usr/lib/libcoretls.dylib
    0x7fff5db52000 -     0x7fff5db53ff3  libcoretls_cfhelpers.dylib (155.220.1) <0959B3E9-6643-3589-8BB3-21D52CDF0EF1> /usr/lib/libcoretls_cfhelpers.dylib
    0x7fff5dfff000 -     0x7fff5e055ff7  libcups.2.dylib (462.10) <2AC7B8C3-0D25-3E0F-A34A-94658A06FF94> /usr/lib/libcups.2.dylib
    0x7fff5e189000 -     0x7fff5e189fff  libenergytrace.dylib (17.200.1) <80BB567A-FD18-3497-BF97-353F57D98CDD> /usr/lib/libenergytrace.dylib
    0x7fff5e1bb000 -     0x7fff5e1c0ff7  libgermantok.dylib (17.15.2) <E5F0F794-FF27-3D64-AE52-C78C6A84DD67> /usr/lib/libgermantok.dylib
    0x7fff5e1c1000 -     0x7fff5e1c6ff7  libheimdal-asn1.dylib (520.260.1) <DC44D1D1-D3DF-38B3-AFC3-1D50B92BE401> /usr/lib/libheimdal-asn1.dylib
    0x7fff5e1f1000 -     0x7fff5e2e1fff  libiconv.2.dylib (51.200.6) <2047C9B7-3F74-3A95-810D-2ED8F0475A99> /usr/lib/libiconv.2.dylib
    0x7fff5e2e2000 -     0x7fff5e543fff  libicucore.A.dylib (62135.0.1) <2B292561-798A-381C-AA41-90EE6549C7CF> /usr/lib/libicucore.A.dylib
    0x7fff5e590000 -     0x7fff5e591fff  liblangid.dylib (128.15.1) <22D05C4F-769B-3075-ABCF-44A0EBACE028> /usr/lib/liblangid.dylib
    0x7fff5e592000 -     0x7fff5e5aaff3  liblzma.5.dylib (10.200.3) <E1F4FD60-1CE4-37B9-AD95-29D348AF1AC0> /usr/lib/liblzma.5.dylib
    0x7fff5e5c2000 -     0x7fff5e666ff7  libmecab.1.0.0.dylib (779.24.1) <A8D0379B-85FA-3B3D-89ED-5CF2C3826AB2> /usr/lib/libmecab.1.0.0.dylib
    0x7fff5e667000 -     0x7fff5e86bfff  libmecabra.dylib (779.24.1) <D71F71E0-30E2-3DB3-B636-7DE13D51FB4B> /usr/lib/libmecabra.dylib
    0x7fff5ea43000 -     0x7fff5ed94ff7  libnetwork.dylib (1229.250.15) <72C7E9E3-B2BE-3300-BE1B-64606222022C> /usr/lib/libnetwork.dylib
    0x7fff5ee26000 -     0x7fff5f5abfdf  libobjc.A.dylib (756.2) <7C312627-43CB-3234-9324-4DEA92D59F50> /usr/lib/libobjc.A.dylib
    0x7fff5f5bd000 -     0x7fff5f5c1ffb  libpam.2.dylib (22.200.1) <586CF87F-349C-393D-AEEB-FB75F94A5EB7> /usr/lib/libpam.2.dylib
    0x7fff5f5c4000 -     0x7fff5f5f9fff  libpcap.A.dylib (79.250.1) <C0893641-7DFF-3A33-BDAE-190FF54837E8> /usr/lib/libpcap.A.dylib
    0x7fff5f712000 -     0x7fff5f72affb  libresolv.9.dylib (65.200.2) <893142A5-F153-3437-A22D-407EE542B5C5> /usr/lib/libresolv.9.dylib
    0x7fff5f72c000 -     0x7fff5f767ff3  libsandbox.1.dylib (851.260.2) <5BB0426C-6DE4-3E12-B6B6-35D4FC318A89> /usr/lib/libsandbox.1.dylib
    0x7fff5f77d000 -     0x7fff5f95afe7  libsqlite3.dylib (274.25) <55CE34C4-00C0-3844-B7ED-80FA7F0AF03F> /usr/lib/libsqlite3.dylib
    0x7fff5fb73000 -     0x7fff5fb76ff7  libutil.dylib (51.200.4) <CE9B18C9-66ED-32D4-9D29-01F8FCB467B0> /usr/lib/libutil.dylib
    0x7fff5fb77000 -     0x7fff5fb84fff  libxar.1.dylib (417.1) <39CCF46B-C81A-34B1-92A1-58C4E5DA846E> /usr/lib/libxar.1.dylib
    0x7fff5fb89000 -     0x7fff5fc6bff3  libxml2.2.dylib (32.8) <54F896CF-6D83-300B-9CD9-1636BE5375A4> /usr/lib/libxml2.2.dylib
    0x7fff5fc6c000 -     0x7fff5fc94ff3  libxslt.1.dylib (16.1) <31F7018B-D8F2-36ED-A13B-1E5A16DFAAA1> /usr/lib/libxslt.1.dylib
    0x7fff5fc95000 -     0x7fff5fca7ff7  libz.1.dylib (70.200.4) <B048FC1F-058F-3A08-A1FE-81D5308CB3E6> /usr/lib/libz.1.dylib
    0x7fff6048b000 -     0x7fff6048fff3  libcache.dylib (81) <1987D1E1-DB11-3291-B12A-EBD55848E02D> /usr/lib/system/libcache.dylib
    0x7fff60490000 -     0x7fff6049aff3  libcommonCrypto.dylib (60118.250.2) <1765BB6E-6784-3653-B16B-CB839721DC9A> /usr/lib/system/libcommonCrypto.dylib
    0x7fff6049b000 -     0x7fff604a2ff7  libcompiler_rt.dylib (63.4) <5212BA7B-B7EA-37B4-AF6E-AC4F507EDFB8> /usr/lib/system/libcompiler_rt.dylib
    0x7fff604a3000 -     0x7fff604acff7  libcopyfile.dylib (146.250.1) <98CD00CD-9B91-3B5C-A9DB-842638050FA8> /usr/lib/system/libcopyfile.dylib
    0x7fff604ad000 -     0x7fff60531fc3  libcorecrypto.dylib (602.260.2) <BED0F385-D8A2-334A-BD7B-0673B5DABBC5> /usr/lib/system/libcorecrypto.dylib
    0x7fff605b8000 -     0x7fff605f1ff7  libdispatch.dylib (1008.250.7) <26DF5B1E-A388-38EF-B73B-AF0E93FB02DB> /usr/lib/system/libdispatch.dylib
    0x7fff605f2000 -     0x7fff6061eff7  libdyld.dylib (655.1.1) <002418CC-AD11-3D10-865B-015591D24E6C> /usr/lib/system/libdyld.dylib
    0x7fff6061f000 -     0x7fff6061fffb  libkeymgr.dylib (30) <0D0F9CA2-8D5A-3273-8723-59987B5827F2> /usr/lib/system/libkeymgr.dylib
    0x7fff60620000 -     0x7fff6062cff3  libkxld.dylib (4903.261.4) <71CF1ECD-AC4E-3FDD-BEDF-14933D843EEE> /usr/lib/system/libkxld.dylib
    0x7fff6062d000 -     0x7fff6062dff7  liblaunch.dylib (1336.261.2) <BD5CDB35-2BB4-349A-8D14-78E387563F89> /usr/lib/system/liblaunch.dylib
    0x7fff6062e000 -     0x7fff60633fff  libmacho.dylib (927.0.2) <3C5C9024-45FD-38C2-B01A-07A322966063> /usr/lib/system/libmacho.dylib
    0x7fff60634000 -     0x7fff60636ffb  libquarantine.dylib (86.220.1) <6D0BC770-7348-3608-9254-F7FFBD347634> /usr/lib/system/libquarantine.dylib
    0x7fff60637000 -     0x7fff60638ff7  libremovefile.dylib (45.200.2) <9FBEB2FF-EEBE-31BC-BCFC-C71F8D0E99B6> /usr/lib/system/libremovefile.dylib
    0x7fff60639000 -     0x7fff60650ff3  libsystem_asl.dylib (356.200.4) <A62A7249-38B8-33FA-9875-F1852590796C> /usr/lib/system/libsystem_asl.dylib
    0x7fff60651000 -     0x7fff60651ff7  libsystem_blocks.dylib (73) <A453E8EE-860D-3CED-B5DC-BE54E9DB4348> /usr/lib/system/libsystem_blocks.dylib
    0x7fff60652000 -     0x7fff606d9fff  libsystem_c.dylib (1272.250.1) <7EDACF78-2FA3-35B8-B051-D70475A35117> /usr/lib/system/libsystem_c.dylib
    0x7fff606da000 -     0x7fff606ddffb  libsystem_configuration.dylib (963.260.1) <C7D2BA09-8D5B-3393-8D84-304D76C55DEF> /usr/lib/system/libsystem_configuration.dylib
    0x7fff606de000 -     0x7fff606e1ff7  libsystem_coreservices.dylib (66) <719F75A4-74C5-3BA6-A09E-0C5A3E5889D7> /usr/lib/system/libsystem_coreservices.dylib
    0x7fff606e2000 -     0x7fff606e8fff  libsystem_darwin.dylib (1272.250.1) <EC9B39A5-9592-3577-8997-7DC721D20D8C> /usr/lib/system/libsystem_darwin.dylib
    0x7fff606e9000 -     0x7fff606efff7  libsystem_dnssd.dylib (878.260.1) <64B79B01-B1B0-3C7E-87A1-023B71843F1F> /usr/lib/system/libsystem_dnssd.dylib
    0x7fff606f0000 -     0x7fff6073bffb  libsystem_info.dylib (517.200.9) <D09D5AE0-2FDC-3A6D-93EC-729F931B1457> /usr/lib/system/libsystem_info.dylib
    0x7fff6073c000 -     0x7fff60764ff7  libsystem_kernel.dylib (4903.261.4) <7B0F52C4-4CC9-3282-8457-C18C6FE1B99E> /usr/lib/system/libsystem_kernel.dylib
    0x7fff60765000 -     0x7fff607b0ff7  libsystem_m.dylib (3158.200.7) <F19B6DB7-014F-3820-831F-389CCDA06EF6> /usr/lib/system/libsystem_m.dylib
    0x7fff607b1000 -     0x7fff607dbfff  libsystem_malloc.dylib (166.261.1) <4D1CEF38-DE32-3ECF-9F70-0ADDD5C8775F> /usr/lib/system/libsystem_malloc.dylib
    0x7fff607dc000 -     0x7fff607e6ff7  libsystem_networkextension.dylib (767.250.2) <FF06F13A-AEFE-3A27-A073-910EF78AEA36> /usr/lib/system/libsystem_networkextension.dylib
    0x7fff607e7000 -     0x7fff607eefff  libsystem_notify.dylib (172.200.21) <145B5CFC-CF73-33CE-BD3D-E8DDE268FFDE> /usr/lib/system/libsystem_notify.dylib
    0x7fff607ef000 -     0x7fff607f8fef  libsystem_platform.dylib (177.250.1) <16949870-9152-3B0D-9EF0-40FAF84B0F1E> /usr/lib/system/libsystem_platform.dylib
    0x7fff607f9000 -     0x7fff60803ff7  libsystem_pthread.dylib (330.250.2) <2D5C08FF-484F-3D59-9132-CE1DCB3F76D7> /usr/lib/system/libsystem_pthread.dylib
    0x7fff60804000 -     0x7fff60807ff7  libsystem_sandbox.dylib (851.260.2) <9CACC5F5-3859-3A1F-BCC6-96DDD4A556A8> /usr/lib/system/libsystem_sandbox.dylib
    0x7fff60808000 -     0x7fff6080aff3  libsystem_secinit.dylib (30.260.2) <88F3CE61-5FAD-3CDA-AA13-B5B18B3EBD26> /usr/lib/system/libsystem_secinit.dylib
    0x7fff6080b000 -     0x7fff60812ff3  libsystem_symptoms.dylib (820.267.1) <03F1C2DD-0F5A-3D9D-88F6-B26C0F94EB52> /usr/lib/system/libsystem_symptoms.dylib
    0x7fff60813000 -     0x7fff60828fff  libsystem_trace.dylib (906.260.1) <100013AE-0443-3CF0-AC17-5D934608B60B> /usr/lib/system/libsystem_trace.dylib
    0x7fff6082a000 -     0x7fff6082fffb  libunwind.dylib (35.4) <24A97A67-F017-3CFC-B0D0-6BD0224B1336> /usr/lib/system/libunwind.dylib
    0x7fff60830000 -     0x7fff6085ffff  libxpc.dylib (1336.261.2) <9A0FFA79-082F-3293-BF49-63976B073B74> /usr/lib/system/libxpc.dylib

Model: MacBookPro15,2, BootROM 220.260.170.0.0 (iBridge: 16.16.5125.0.0,0), 4 processors, Intel Core i5, 2.3 GHz, 8 GB, SMC 
Graphics: kHW_IntelIrisGraphics655Item, Intel Iris Plus Graphics 655, spdisplays_builtin
Memory Module: BANK 0/ChannelA-DIMM0, 4 GB, LPDDR3, 2133 MHz, Samsung, K4E6E304EC-EGCG
Memory Module: BANK 2/ChannelB-DIMM0, 4 GB, LPDDR3, 2133 MHz, Samsung, K4E6E304EC-EGCG
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x7BF), wl0: Mar 28 2019 19:17:52 version 9.137.9.0.32.6.34 FWID 01-36f56c94
Bluetooth: Version 6.0.12f1, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
USB Device: USB 3.1 Bus
USB Device: Quest
USB Device: iBridge Bus
USB Device: iBridge DFR brightness
USB Device: iBridge Display
USB Device: Apple Internal Keyboard / Trackpad
USB Device: Headset
USB Device: iBridge ALS
USB Device: iBridge FaceTime HD Camera (Built-in)
USB Device: iBridge
Thunderbolt Bus: MacBook Pro, Apple Inc., 41.2
Thunderbolt Bus: MacBook Pro, Apple Inc., 41.2

**Comment by waspFk** [comment]
Hello, sorry i have a same issue as @zaccolley : 

System.ApplicationException: Trying to patch different version of code at LibSaberPatch.Apk.PatchSignatureCheck() in C:\projects\QuestSaberPatch\LibSaberPatch\Apk.cs:line 97 at jsonApp.Program.RunInvocation(Invocation inv) in C:\projects\QuestSaberPatch\jsonApp\Program.cs:line 63

I try unistall and install again but it's not working
Do i need to do something special ?

**Comment by slider162** [comment]
Running the update, I get a ffmpeg.dll was not found.

![image](https://user-images.githubusercontent.com/5995195/58918032-36b9de80-86ee-11e9-9486-628837ff69d7.png)

**Comment by ATechAdventurer** [comment]
I updated that release to include ffmpeg.dll @slider162 @waspFk @lazerwalker

**Comment by rafikiki** [comment]
Should this alpha release allow custom songs to be loaded into 1.0.1? If not, does anyone have a backup of 1.0.0 I can use, or is that not possible?

**Comment by amydionne** [comment]
The only time I wish I had Windows based computer....

**Comment by Purplexgd** [comment]
It doesn’t work on Windows either, it’s missing a directory, that’s why it won’t work. They’re fixing it now.

**Comment by slider162** [comment]
Got past the ffmpeg error.  Now just getting "FailError: Failure: 'open failed: No such file or directory'."

![image](https://user-images.githubusercontent.com/5995195/58922442-7984b200-8700-11e9-9eba-6787dfa7f49f.png)

**Comment by ATechAdventurer** [comment]
I have been trying to figure out where this issue came from all day I'll keep looking

**Comment by SaladFork** [comment]
Looks like the Mac one needs to be updated too?
```bash
$ wget https://github.com/the-expanse/SideQuest/releases/download/0.2.1-alpha.1/SideQuest-darwin-x64.zip
$ md5 SideQuest-darwin-x64.zip
MD5 (SideQuest-darwin-x64.zip) = 1b45bb899ee6bce0933fbf74c4bb30a8
$ open SideQuest-darwin-x64.zip
$ open SideQuest-darwin-x64/SideQuest.app
```

```
Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    DYLD, [0x1] Library missing

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: @rpath/libffmpeg.dylib
  Referenced from: /private/var/folders/52/hyxcdg2j2blcm46gy3lzwd_40000gn/T/AppTranslocation/DE98DAC5-D64D-4D63-B658-CD120631F974/d/SideQuest.app/Contents/Frameworks/Electron Framework.framework/Electron Framework
  Reason: image not found
```

**Comment by slider162** [comment]
> I have been trying to figure out where this issue came from all day I'll keep looking

I went in and played a round.  Got past the "No such directory" and ran into this:

System.ApplicationException: Trying to patch different version of code at LibSaberPatch.Apk.PatchSignatureCheck() in C:\projects\QuestSaberPatch\LibSaberPatch\Apk.cs:line 97 at jsonApp.Program.RunInvocation(Invocation inv) in C:\projects\QuestSaberPatch\jsonApp\Program.cs:line 63

**Comment by cryptk** [comment]
@slider as mentioned in https://github.com/the-expanse/SideQuest/issues/61#issuecomment-498813823 and https://github.com/the-expanse/SideQuest/issues/61#issuecomment-498844064 we need to wait for QuestSaberPatch to be updated before we can do custom songs on 1.0.1

**Comment by joshlevitre** [comment]
Looks like QuestSaberPatch got an update!

**Comment by shaneharris** [comment]
Please try the latest release!
Thanks!
S

**Comment by slider162** [comment]
Worked for me this morning.  Thanks!!!
